django_fido/models.py

```diff
@@ -104,17 +104,15 @@ def attestation(self, value: AttestationObject):
         self.attestation_data = base64.b64encode(value).decode('utf-8')
         self.credential_id_data = base64.b64encode(value.auth_data.credential_data.credential_id).decode('utf-8')
 
-    def _get_metadata(self) -> Optional['AuthenticatorMetadata']:
-        """Get the appropriate metadata."""
-        # First test the presence of aaguid - FIDO 2
-        if hasattr(self.attestation.auth_data, 'credential_data') and \
-                self.attestation.auth_data.credential_data is not None and \
-                self.attestation.auth_data.credential_data.aaguid != NULL_AAGUID:
-            identifier = str(UUID(b2a_hex(self.credential.aaguid).decode()))
-            try:
-                return AuthenticatorMetadata.objects.get(identifier=identifier)
-            except AuthenticatorMetadata.DoesNotExist:
-                return None
+    @cached_property
+    def identifier(self) -> Optional[Union[str, bytes]]:
```

Comment:
This can be replaced by `AnyStr`

Reply:
Substituted

Reply:
Apparently a `TypeVar` won't work as there is nothin that can constrain the value of `AnyStr`.